tests/dynamic_sharing_service/policy/dynamic.policy.builder.spec.js

```diff
@@ -210,7 +210,7 @@ describe('DynamicPolicyBuilder', () => {
       .withWantedAuthType(99)
       .build();
 
-    const expectedAuthTypes = [1, 2, 99];
```

Comment:
As discussed, we should try to avoid reusing constants in tests - we can redefine them in the test to make the test easier to follow